docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/classrooms_core.module

```diff
@@ -198,6 +199,16 @@ function classrooms_core_preprocess_block(&$variables) {
  */
 function classrooms_core_preprocess_field(&$variables, $hook) {
   switch ($variables['element']['#field_name']) {
+    case 'field_room_type':
+      $link_items = $variables['items'];
+      foreach ($link_items as &$item) {
+        $item['content']['#options']['attributes']['class'][] = 'bttn bttn--primary bttn--light-font bttn--no-text bttn--small';
+        $item['content']['#title'] = Markup::create('<i role="presentation" class="fa-building fas"></i>&nbsp; <span class="field__label">Room Type:</span>&nbsp; ' . $item['content']['#title']);
+      }
+      $variables['items'] = $link_items;
+      $variables['items'][0]['content']['#url'] = Url::fromRoute('entity.node.canonical', ['node' => 206]);
```

Comment:
Why this value is hard-coded?

Reply:
I'm trying to get the field label and icon rendered within the link.  Do you have a suggestion on how we can do that?

Reply:
This has been removed and moved to the template.